make_queue/forms.py

```diff
@@ -211,7 +222,6 @@ class Meta:
 
 
 class EditMachineForm(BaseMachineForm):
-    machine_type = None
 
     class Meta(BaseMachineForm.Meta):
-        exclude = ["machine_type", "machine_model"]
+        exclude = ["machine_model"]
```

Comment:
The machine type is included, so that the javascript that disables `stream_name` works correctly in the `EditMachineView`. Is there a point of excluding `machine_model`? I don't see the harm in leaving it in as well.

Reply:
`machine_model` was removed from the `exclude` field in #351 (more specifically d133cd8f9e5b3d41c365b68566b0dd7c59d66c4e), so this is resolved 😄